server/protocols/jmap-rfc-8621/doc/specs/spec/quotas/quota.mdown

```diff
@@ -79,10 +79,9 @@ A **FilterCondition** object has the following properties, any of which may be o
 A Quota object matches the FilterCondition if and only if all of the given conditions match. If zero properties are
 specified, it is automatically true for all objects.
 
-The following Quota properties MUST be supported for sorting:
+The FilterOperator (AND/OR/NOT) is not supported
 
-* **name**
-* **used**
+The following properties are not supported: sort, position, anchor, anchorOffset, limit, calculateTotal.
```

Comment:
You modified the spec here! That is bad!

```
The following Quota properties MUST be supported for sorting:

* **name**
* **used**

> :warning:
> Not implemented
> The following properties are not supported: sort, position, anchor, anchorOffset, limit, calculateTotal.
```

BTW we should likely support the `sort` option with the two mentionned properties...

Reply:
> BTW we should likely support the sort option with the two mentionned properties...

We don't support `limit, offset`, so `sort` don't make sense.
The list returns only `ids` value in the array json

Reply:
Ok but do not modify the spec, annotate it!